# list of ids of tag
int32[] id

# list of tag position vector
float32[] position

# list of tag rotation matrix
float32[] rotation           
